atmo/clusters/tasks.py

```diff
@@ -0,0 +1,12 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The terminology for RQ is jobs and not tasks, mind renaming the file to `jobs.py`?